bin/generate_corpora.xq

```diff
@@ -0,0 +1,61 @@
+xquery version "3.0" encoding "UTF-8";
+
+declare namespace local="http://library.princeton.edu";
+declare namespace tei="http://www.tei-c.org/ns/1.0";
+declare namespace dc = "http://purl.org/dc/elements/1.1/";
+declare namespace dcterms="http://purl.org/dc/terms/";
+declare namespace gettyterms="http://portal.getty.edu/terms/gri/";
+
+declare function local:docs-from-dclnum($dclnum) {
+    let $hits := collection('/db/apps/cicognara/data/getty_tei')//tei:idno[@type= 'dcl'][.=$dclnum]
+    for $hit in $hits return $hit/ancestor::tei:TEI
+};
+
+let $target_directory := "/tmp/corpora/"
```

Comment:
Is there any reason to use the system temp directory instead of having one in this project?

Reply:
https://github.com/pulibrary/cicognara-catalogo/issues/167